### PR TITLE
Handle null date of birth in individual

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java [hunk 1]
```diff
@@ -33,9 +33,12 @@ public CCDIndividual to(Individual individual) {
         individual.getRepresentative()
             .ifPresent(representative -> builder.representative(representativeMapper.to(representative)));
 
+        if (individual.getDateOfBirth() != null) {
+            builder.dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE));
+        }
+
         return builder
             .name(individual.getName())
-            .dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE))
```
Comment (singh-surjeet) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java]: will it not fail again?

Comment (dharmendrak) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java]: removed here and doing null check ad adding

File: ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java [hunk 2]
```diff
@@ -33,9 +33,12 @@ public CCDIndividual to(Individual individual) {
         individual.getRepresentative()
             .ifPresent(representative -> builder.representative(representativeMapper.to(representative)));
 
+        if (individual.getDateOfBirth() != null) {
+            builder.dateOfBirth(individual.getDateOfBirth().format(DateTimeFormatter.ISO_DATE));
```
Comment (geek-soft) [inline ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java]: DOB should be optional when nullable.

Comment (geek-soft) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java]: Also create test case for to prevent failures in the future

Comment (singh-surjeet) [reply ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/IndividualMapper.java]: sure. Will do

**Pull request merged**